demisto_sdk/commands/init/initiator.py

```diff
@@ -232,6 +234,7 @@ def pack_init(self) -> bool:
             fill_manually = user_response in ['y', 'yes']
 
             pack_metadata = Initiator.create_metadata(fill_manually)
+            self.category = pack_metadata['categories'][0]
```

Comment:
not sure if pack categories are the same as integration categories - need to check that - you can look at the validations to make sure.